src/main/java/net/minecraftforge/common/extensions/IForgeGameTestHelper.java

```diff
@@ -51,7 +51,7 @@ public boolean isCreative() {
         var channel = new EmbeddedChannel(connection);
         channel.attr(Connection.ATTRIBUTE_SERVERBOUND_PROTOCOL).set(ConnectionProtocol.PLAY.codec(PacketFlow.SERVERBOUND));
         // This sets the connection/listener in the player
-        new ServerGamePacketListenerImpl(level.getServer(), connection, player, cookie, player);
+        new ServerGamePacketListenerImpl(level.getServer(), connection, player, cookie/*, player*/); //Ketting - fix compilation error. IDK if this is the proper fix and IDC. We don't use Forge's TestServer anyway.
```

Comment:
this was a coding error made while patching, the fifth value does not exist on both bukkit and forge, you can just remove it